woocommerce/woocommerce-gutenberg-products-block
# Set up Jest for JS testing

This PR adds a Jest setup, commands, and a single component test to show it's working (it's not the most useful test, but other components are wrapped in API calls).

This PR also adds the commands `test:update` to update snapshots and `test:watch` to watch for changes and re-run tests.

**To test**

1. Run `npm test` to run all tests (only one, currently)
2. Expect: the tests should run, and pass.
